plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -50,6 +60,14 @@ var sampleConfig = `
 
   ## Namespace for the CloudWatch MetricDatums
   namespace = "InfluxData/Telegraf"
+
+  ## If you have a large amount of metrics, you should consider to send 
+  ## statistic values instead of raw metrics. This would not only improve
+  ## performance but also save AWS API cost. Use basicstats aggregator to
+  ## calculate required statistic fields (count, min, max, and sum) and 
+  ## enable this flag. This plugin would try to parse those fields and 
+  ## send statistic values to Cloudwatch.
+  # enable_statistic_values = false
```

Comment:
Let's call this option `write_statistics`, this is little shorter, but also communicates that it is something that affects the output data.  So long as you are in here, can you mention in the comment that if a field does not have the required statistic fields then it will still be sent as a custom/raw metric.

Reply:
okay, will update the field name and comments